src/phoebus_driver.cpp

```diff
@@ -605,6 +807,7 @@ parthenon::Packages_t ProcessPackages(std::unique_ptr<ParameterInput> &pin) {
   packages.Add(Microphysics::EOS::Initialize(pin.get()));
   packages.Add(Microphysics::Opacity::Initialize(pin.get()));
   packages.Add(Geometry::Initialize(pin.get()));
+  // packages.Add(radiation::Initialize(pin.get()));
```

Comment:
is there no radiation package anymore?

Reply:
I moved it to below `fluid` in the hopes that this would change the order in which per-package `FillDerived` calls were called (we need to update rad after fluid because rad prims are in the fluid frame) but it doesn't seem to. I'll delete the commented out extra line.

Reply:
The radiation package still exists and is initialized two lines down, I just forgot to uncomment the original place where I removed it

Reply:
Ah ok :+1: